src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message in the `doWork` method should be more informative. According to the checklist, it should indicate the type of machine that is working. Consider specifying that it is a Truck, for example: 'Truck is doing its work'.